src/MediaWiki/EditInfo.php

```diff
@@ -44,10 +44,10 @@ class EditInfo {
 	 * @since 1.9
 	 *
 	 * @param WikiPage $page
+	 * @param User $user
```

Comment:
This docblock is redundant. Should we remove the whole thing, or keep only `@since`?

Reply:
Both are fine IMO. Definitely good to get rid of the redundant docs. I'd probably kill the whole thing, as these @since is not very useful here